include/mruby/gc.h

```diff
@@ -48,7 +48,6 @@ typedef struct mrb_heap_page {
   struct mrb_heap_page *free_next;
   struct mrb_heap_page *free_prev;
   mrb_bool old:1;
-  void *objects[];
```

Comment:
If it is simply deleted, the data structure will be difficult to understand.
I think it's easier to understand if you comment it out and add more annotations.